FWCore/ServiceRegistry/src/GlobalContext.cc

```diff
@@ -118,6 +120,15 @@ namespace edm {
     }
   }
 
+  void exceptionContext(cms::Exception& ex, GlobalContext const& globalContext, char const* context) {
+    std::ostringstream ost;
+    if (context && *context != '\0') {
+      ex.addContext(context);
```

Comment:
Maybe we should think of adding `std::string_view` overload for the `addContext()` (but not in this PR).